src/app.ts

```diff
@@ -184,7 +184,7 @@ export async function getApp(
   await app.register(fastifyJWT, {
     secret: configOverrides.jwtKeys ?? {
       private: '-', // Private key blank, as this service won't create JWT tokens, only verify them
-      public: appConfig.jwtPublicKey,
+      public: appConfig.jwtPublicKey.replaceAll('||', '\n'),
```

Comment:
can you add a test for this? AFAIR ConfigScope has refresh method, so you can adjust process.env, refresh configscope and resolve new copy of a config

Reply:
oh, this is in app
maybe we should move this logic to config.ts?

Reply:
Maybe config.ts would be even a better place

Reply:
Rewrote a bit